src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+abstract class Figure implements AreaCalculator {
+    public static final int VALUE_LIMIT = 50;
```

Comment:
this field also will be redundant, after we remove `Random` and `ColorSupplier` fields